base/interactiveutil.jl

```diff
@@ -571,11 +571,10 @@ end
 downloadcmd = nothing
 if is_windows()
     function download(url::AbstractString, filename::AbstractString)
-        res = ccall((:URLDownloadToFileW,:urlmon),stdcall,Cuint,
-                    (Ptr{Void},Cwstring,Cwstring,Cuint,Ptr{Void}),C_NULL,url,filename,0,C_NULL)
-        if res != 0
-            error("automatic download failed (error: $res): $url")
-        end
+        ps = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
+        enable_tls12 = "[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12"
+        download_cmd = "(New-Object System.Net.Webclient).DownloadFile('$(replace(url, "'", "''"))', '$(replace(filename, "'", "''"))')"
```

Comment:
I think I see your point, but don't you want 
`replace(filename, "'", "\'")`
instead of
`replace(filename, "'", "''")`
or are those the same

and is this only needed here since we first interpolate the string into another string before  interpolating into the cmd  `run(` `)` ? I'm just trying to understand why the non windows download cmd doesn't need this.